Resources/Prototypes/Entities/Mobs/Species/human.yml

```diff
@@ -9,7 +9,10 @@
     starvationDamage:
       types:
         Cold: 0.5
-        Bloodloss: 0.5
+        Bloodloss: 0.5 
+  - type: Icon # It will not have an icon in the adminspawn menu without this. Body parts seem fine for whatever reason.
+    sprite: Mobs/Species/Human/parts.rsi
+    state: full
```

Comment:
Are you sure this is even needed with https://github.com/space-wizards/space-station-14/pull/18291

Reply:
That isn’t merged yet, but you also explicitly told me earlier than the NPCs needed it still even with that PR.

Reply:
Maybe I was thinking of it being on MobHuman but that's probably fine then.